pkg/windows/windows.go

```diff
@@ -496,6 +496,12 @@ func (vm *windows) ConfigureHybridOverlay(nodeName string) error {
 	hybridOverlayServiceArgs := "--node " + nodeName + customVxlanPortArg + " --k8s-kubeconfig c:\\k\\kubeconfig " +
 		"--windows-service " + "--logfile " + hybridOverlayLogDir + "hybrid-overlay.log"
 
+	// check log level and increase hybrid-overlay verbosity if needed
+	if ctrl.Log.V(1).Enabled() {
```

Comment:
Please use windows.log for this check

Reply:
Addressed in [41fa41a ](https://github.com/openshift/windows-machine-config-operator/commit/41fa41ae3430f6f999fe1c15ddd576d204283dee)